test_runner/fixtures/neon_fixtures.py

```diff
@@ -3983,18 +3983,27 @@ def check_restored_datadir_content(test_output_dir: Path, env: NeonEnv, endpoint
 
 def logical_replication_sync(subscriber: VanillaPostgres, publisher: Endpoint) -> Lsn:
```

Comment:
There is `logical_replication_sync` function in fixtures now. Do we actually need one more?

Reply:
I do not duplicate it, just add `pg_subscription_rel` check to it. Forgot to link example of supposedly related flakiness:
https://neon-github-public-dev.s3.amazonaws.com/reports/pr-6988/8112986047/index.html#suites/180444c850d4a41d41eb0a410dc16d84/1ff911895469b1af

Reply:
This change still looks reasonable to me. Let's finish the review and merge? @arssher @knizhnik